games-emulation/higan/higan-116_pre20210404.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop toolchain-funcs xdg
+
+DESCRIPTION="A multi-system emulator focused on accuracy, preservation, and configurability"
+HOMEPAGE="https://github.com/higan-emu/higan"
+SRC_URI="https://github.com/higan-emu/${PN}/archive/44cbb88d49d5a77f19e353fad4bc7fbab6909ca9.tar.gz -> ${P}.tar.gz"
```

Comment:
Please make that use `${COMMIT}` and set `COMMIT=44cbb88d49d5a77f19e353fad4bc7fbab6909ca9` before inherit.
You can use `COMMIT` for setting `S` as well then, and ideally move that line below `SRC_URI` since ultimately here S is linked to SRC_URI.

Reply:
Do you mean move `S` up closer to `SRC_URI` so that `S` is right below it?

Reply:
yep